src/main/app/case/from-api-format.ts

```diff
@@ -148,6 +148,12 @@ const fields: FromApiConverters = {
       applicant2SolicitorAddressCountry: address?.[6],
     };
   },
+  applicant2SolicitorAddressOverseas: ({ applicant2SolicitorAddressOverseas }) => ({
+    applicant2SolicitorAddressOverseas:
+      applicant2SolicitorAddressOverseas === undefined || applicant2SolicitorAddressOverseas === YesOrNo.NO
+        ? YesOrNo.NO
+        : YesOrNo.YES,
```

Comment:
If we only expect applicant2SolicitorAddressOverseas to be `undefined`, `YES`, or `NO`, it might be reasonable to use the nullish coalescing operator (`??`) to fall back on the default value?:
```suggestion
    applicant2SolicitorAddressOverseas: applicant2SolicitorAddressOverseas ?? YesOrNo.NO,
```

Reply:
Absolutely yes, but working in an environment with (primarily) backend Java devs who also have to work on the FE, it's not as easy to understand what it's doing imho (Java does not have an equivalent operator).

Reply:
I've had a chat with Aaron & Del, because I like the change but I wanted some extra opinions regarding the readability.  They're good with it (also, they both understand it now haha).  I'll start with the PR this is based on, and get it merged in.

Reply:
Done